README.md

```diff
@@ -122,7 +122,7 @@ promise, data, etc.
 // args is an object set via the directive in the view.
 // likely it is some object on a controller scope that gives
 // meaning to the endpoint.
-extensionInput.register('endpoint1', function(args) {
+extensionRegistry.register('endpoint1', function(args) {
```

Comment:
Shall we now also update this to `extensionRegistry.add()`?  Seems redundant here lil bit.